Sources/WebSocketKit/WebSocket+Connect.swift

```diff
@@ -20,6 +20,25 @@ extension WebSocket {
         on eventLoopGroup: EventLoopGroup,
         onUpgrade: @Sendable @escaping (WebSocket) -> ()
     ) -> EventLoopFuture<Void> {
+        #if os(iOS) || os(tvOS) || os(watchOS) || os(macOS)
+        let optionalURL: URL?
+        if #available(iOS 17.0, macOS 14.3, tvOS 17.0, watchOS 10.0, *) {
```

Comment:
```suggestion
        if #available(iOS 17.0, macOS 14.0, tvOS 17.0, watchOS 10.0, *) {
```
Should fix the issue